ggml/src/ggml-cuda/ggml-cuda.cu

```diff
@@ -2610,13 +2610,15 @@ static bool is_cuda_graph_update_required(ggml_backend_cuda_context * cuda_ctx,
 
 static void update_cuda_graph_executable(ggml_backend_cuda_context * cuda_ctx) {
 
-    cudaGraphExecUpdateResultInfo result_info;
-#ifdef __HIP_PLATFORM_AMD__
-    hipGraphNode_t errorNode;
-    hipError_t stat = hipGraphExecUpdate(cuda_ctx->cuda_graph->instance, cuda_ctx->cuda_graph->graph, &errorNode, &result_info);
+#if (CUDART_VERSION < 12000 || defined(__HIP_PLATFORM_AMD__))
+    cudaGraphNode_t errorNode;
+    cudaGraphExecUpdateResult result_info;
+    cudaError_t stat = cudaGraphExecUpdate(cuda_ctx->cuda_graph->instance, cuda_ctx->cuda_graph->graph, &errorNode, &result_info);
 #else
+    cudaGraphExecUpdateResultInfo result_info;
```

Comment:
Is this correctly `cudaGraphExecUpdateResultInfo`? Or it should be `cudaGraphExecUpdateResult`?

As it is, `cudaGraphExecUpdateResultInfo` is only defined in `vendors/musa.h`:

https://github.com/ggml-org/llama.cpp/blob/db400a6598337e8a392bc5c77908118efd3506a0/ggml/src/ggml-cuda/vendors/musa.h#L122